src/base/units.h

```diff
@@ -10,8 +10,8 @@
 // This file is part of Cantera. See License.txt in the top-level directory or
 // at https://cantera.org/license.txt for license and copyright information.
 
-#ifndef CT_UNITS_H
-#define CT_UNITS_H
+#ifndef CT_LEGACY_UNITS_H
```

Comment:
Not ever having noticed made me curious. It appears that the docstring is incorrect: `misc.cpp` no longer exists. At this point, it is included in `application.cpp` and `global.cpp`.

Reply:
I deleted that part of the docstring.

Reply:
Out of curiosity - is this only used by CTI/XML or is it more pervasive?